lldb/tools/lldb-dap/package.json

```diff
@@ -175,9 +175,10 @@
                     "type": "array",
                     "description": "Additional environment variables to set when launching the program. E.g. `[\"FOO=1\", \"BAR\"]`",
                     "items": {
-                      "type": "string"
+                      "type": "string",
```

Comment:
try to use something like "anyOf" https://json-schema.org/understanding-json-schema/reference/combining#anyOf, which will make it very obvious that it's a string array or a map

Reply:
It does use anyOf on line 163. the pattern regex is only for array for it to show a warning when it does not match any of the pattern "NO_VALUE", "WITH_VALUE=BAR", "EMPTY_VALUE=", "SPACE=Hello World"